src/app/ddDrakeModel.cpp

```diff
@@ -52,7 +52,7 @@ using std::string;
 using std::istringstream;
 using namespace Eigen;
 
-#if VTK_MAJOR_VERSION == 6
+#if VTK_MAJOR_VERSION >= 6
```

Comment:
It seems we are not trying to keep VTK5 support elsewhere; maybe we should get rid of this?